src/components/RealTimeInterface/SessionStarted.vue

```diff
@@ -226,51 +226,78 @@ async function submitBugReport (bug, payload) {
 }
 
 const sendGoCommand = async () => {
+  if (configurationStore.demo) {
+    realTimeSessionsStore.setPreviousThumbnailCount(realTimeSessionsStore.thumbnailCount)
```

Comment:
I'm pretty confused by the previous / current thumbnail counts. I think maybe its a naming issue causing the confusion? If you want to know how many thumbnails their should be, then keeping track of your requested exposure count should be that number. If you want to know how many thumbnails their currently are, then the query to get the thumbnails and filtering for raw ones should get that number. It's unclear to me where in that you need to keep track of the previous count of the thumbnails?